crates/telio-relay/src/derp/http.rs

```diff
@@ -55,9 +56,12 @@ pub type Protect = Arc<dyn Fn(i32) + Send + Sync + 'static>;
 const MAX_TCP_PACKET_SIZE: usize = u16::MAX as usize;
 
 pub struct DerpConnection {
-    pub comms: Chan<(PublicKey, Vec<u8>)>,
+    pub comms: Chan<(Option<PublicKey>, Vec<u8>)>,
```

Comment:
Yep this is for derp module -> derp connection communication, so since we don't have public key in messages directly with derp, made this into optional